dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/Microsoft.Windows.ApplicationModel.Resources.vcxproj

```diff
@@ -164,7 +164,7 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="..\..\..\..\..\packages\Microsoft.FrameworkUdk\$(FrameworkUdkVersion)\build\native\Microsoft.FrameworkUdk.targets" Condition="Exists('..\..\..\..\..\packages\Microsoft.FrameworkUdk\$(FrameworkUdkVersion)\build\native\Microsoft.FrameworkUdk.targets')" />
+    <Import Project="..\..\..\..\..\packages\Microsoft.FrameworkUdk\$(MicrosoftFrameworkUdkPackageVersion)\build\native\Microsoft.FrameworkUdk.targets" Condition="Exists('..\..\..\..\..\packages\Microsoft.FrameworkUdk\$(MicrosoftFrameworkUdkPackageVersion)\build\native\Microsoft.FrameworkUdk.targets')" />
```

Comment:
MicrosoftFrameworkUdkPackageVersion should use the redirect name "FrameworkUdkVersion"

If not, we should remove FrameworkUdkVersion from version.props.

Reply:
Yup. No more aliases. If it's a ProductDependency then it's Foo.Bar = macro FooBarPackageVersion and if it's a ToolsetDependency = macro FooBarVersion

But it's always FooBar. Even if it's Microsoft.Foo.Bar, macro = MicrosoftFooBarVersion (or MicrosoftFooBarPackageVersion). No more "drop the leading 'Microsoft' _sometimes_". See Version.Details.xml for the simple macro naming rules we now always follow